third_party/clevr_robot_env_utils/generate_question.py

```diff
@@ -15,7 +15,7 @@
 
 import json
 import os
-import random
+# import random
```

Comment:
Could just delete this line if we're not using it anymore?

Reply:
yep!